### PR TITLE
Renamed MergeWindowedOperator to WindowedMergeOperator.

### PR DESCRIPTION
**Pull request by ShunxinLu**
@davidyan74 @tweise Please review.

### CODE REVIEW

File: library/src/test/java/org/apache/apex/malhar/lib/window/sample/Merge/KeyedWindowedMergeOperatorTestApplication.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.apex.malhar.lib.window.impl;
+package org.apache.apex.malhar.lib.window.sample.Merge;
```
Comment (tweise) [inline library/src/test/java/org/apache/apex/malhar/lib/window/sample/Merge/KeyedWindowedMergeOperatorTestApplication.java]: package names should be lowercase

**Review by tweise — CHANGES_REQUESTED** [review]

**Comment by tweise** [comment]
@davidyan74 shouldn't these "tests" that look more like examples be moved under demos and fixed to actually run in the CI. Then we could also point to them as examples in the documentation.

**Comment by davidyan74** [comment]
@tweise I don't have preferences either way. But should that be separate from this PR?

**Comment by davidyan74** [comment]
@ShunxinLu please just change the package name. The location of "tests" should be separate and I can just work on that after this is merged

**Comment by ShunxinLu** [comment]
@davidyan74 @tweise The name of package is renamed to lowercases. Please take a look.

**Comment by tweise** [comment]
Please rebase to HEAD of master.

Please also remove the renames for the sample apps and create a JIRA to provide them as examples under demos.

```
rename library/src/test/java/org/apache/apex/malhar/lib/window/{impl => sample/merge}/KeyedWindowedMergeOperatorTestApplication.java (95%)
 rename library/src/test/java/org/apache/apex/malhar/lib/window/{impl => sample/merge}/WindowedMergeOperatorTestApplication.java (96%)
```

**Comment by ShunxinLu** [comment]
@tweise Does it mean that I only change the name of the interface in this PR? And move the sample apps under demos in another PR?

**Comment by tweise** [comment]
Yes, that's my suggestion. It would be great if you could pick up the demos work!

**Pull request merged**